.github/workflows/tag-and-build.yml

```diff
@@ -24,6 +24,10 @@ on:
         description: 'Published version of InstaScale'
         required: true
         default: 'v0.0.0-dev'
+      kuberay-version:
+        description: 'Published version of KubeRay'
```

Comment:
We don't publish a KubeRay version strictly speaking. So maybe we can describe it as "Tested version of KubeRay"?

Reply:
Updated the description